docs/troubleshooting.md

```diff
@@ -22,6 +22,130 @@ will prompt you for upgrade using a notification. To check manually, run
+happens when the installed version of Git does not match the one that was
+included in the VFS for Git installer. Please download and install the
+matching version of Git from
+[the VFS for Git releases page](https://github.com/microsoft/vfsforgit/releases).
+
+### 404 Errors, or "The Git repository with name or identifier X does not exist..."
+
+If your `gvfs clone <url>` command fails with this error, then check if you
+can access `<url>` in a web browser. If you cannot see the repository in the
+web, then you do not have permissions to read that repository. These issues
+cannot be resolved by the VFS for Git team and must be done by your repository
+administrators.
+
+If you _can_ see the repository in the web, then likely you have a stale
+credential in your credential manager that needs updating. VFS for Git
+_should_ attempt to renew your credential. If it does not, then go to
+Windows Credential Manager and delete the Git credential for that URL.
+
+### ProjFS Installation Issue
+
+**Symptoms:**
+
+- VFS for Git will not mount
+- The mount logs (or mount CLI) have a ProjFS related error.
+Examples:
+   - "Service prjflt was not found on computer".
+    - "Could not load file or assembly 'ProjectedFSLib.Managed.dll' or one
+      of its dependencies. The specified module could not be found."
+    - "Attaching the filter resulted in: 2149515283"
+    - "StartVirtualizationInstance failed: 80070057(InvalidArg)"
+
+**Fix:**
+
+The easiest way to fix ProjFS issues is to completely remove ProjFS, and 
+then rely on `GVFS.Service` to re-install (or re-enable) ProjFS.
+
+1. If `C:\Program Files\GVFS\ProjectedFSLib.dll` is present, delete it
+2. Determine if inbox ProjFS is currently enabled:
+
+*In an Administrator Command Prompt*
+
+```powershell -NonInteractive -NoProfile -Command "&{Get-WindowsOptionalFeature -Online -FeatureName Client-ProjFS}"```
+
+Check the value of `State:` in the output to see if inbox ProjFS is enabled.
+
+3. If ProjFS **is enabled**
+   - Restart machine
+   - Attempt to mount
+   - If mount fails, manually disable ProjFS:
+       - Control Panel->Programs->Turn Windows features on or off
+       - Uncheck "Windows Projected File System"
+      - Click OK
+      - Restart machine
+      - (When `GVFS.Service` starts, it will automatically re-enable the optional feature)
+    - `gvfs mount` repo
+    - If the mount still fails, restart one more time and try `gvfs mount` again
+
+4. If ProjFS **is not enabled**
+    - Manually remove ProjFS and GvFlt
+    From an Administrator command prompt:
+        - `sc stop GVFS.Service`
+       -  `sc stop gvflt`
+       -  `sc delete gvflt`
+       - `sc stop prjflt`
+      - `sc delete prjflt`
+      - `del c:\Windows\System32\drivers\prjflt.sys`
+      - `sc start GVFS.Service`
+    - `gvfs mount` repo
+
+If the above steps do not resolve the issue, and the user is on Windows
+Server, ensure they have the latest version of GVFS installed.

```

Comment:
is there a link we could add here pointing to the latest version download page?
maybe a side-bar showing the output of `git version ; gvfs version ; ....`

Reply:
I'll link to the releases page to avoid needing to update this doc with every release.